memfault-ecs-cli/meta.yaml

```diff
@@ -0,0 +1,22 @@
+{% set name = "ecs-cli" %}
+{% set version = "1.21.0.mflt" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/memfault/memfault-ecs-cli/releases/download/{{ version }}/ecs-cli.darwin-amd64 # [osx]
+  sha256: 13c631695ada3d14ecdeb6ea4896ab4e94bf9e3a6f574b2ed57596643f0db496 # [osx]
+  url: https://github.com/memfault/memfault-ecs-cli/releases/download/{{ version }}/ecs-cli.linux-amd64 # [linux64]
+  sha256: 80f7eee721e1b0268173c8a5a1400486ddae409d737b6c0f4b5fd58baa37e2bb # [linux64]
+
+build:
+  string: '0'
```

Comment:
This is totally new to me, but looking at the docs (https://docs.conda.io/projects/conda-build/en/stable/resources/define-metadata.html#build-section) and https://github.com/memfault/conda-recipes/blob/e24613ebd89abaa8c3db6ceadb095b8dc41e8fea/terraform/meta.yaml maybe this should be...

```suggestion
build:
  number: 0
```

Reply:
With the caveat that I'm completely unsure as to what the implications of either are, we're pretty inconsistent w/ the number vs string use in this repo. There's a very slight majority of em specifying string based on my _super scientific_ grep method of counting.
<img width="697" alt="Screenshot 2023-11-21 at 2 46 42 PM" src="https://github.com/memfault/conda-recipes/assets/7935795/75de4ff8-8fe4-4ffa-a897-7cd2a351d673">
<img width="655" alt="Screenshot 2023-11-21 at 2 47 02 PM" src="https://github.com/memfault/conda-recipes/assets/7935795/5db65347-040c-4603-83ce-d17bc64810f7">